packages/server/package.json

```diff
@@ -45,13 +46,9 @@
     "graphql-tools": "^4.0.7",
     "ioredis": "^4.19.4",
     "knex": "^1.0.3",
-    "lodash.chunk": "^4.2.0",
-    "lodash.debounce": "^4.0.8",
-    "lodash.get": "^4.4.2",
-    "lodash.merge": "^4.6.2",
-    "lodash.set": "^4.3.2",
-    "lodash.values": "^4.3.0",
+    "lodash": "^4.17.21",
```

Comment:
For the server-side it's not necessary to install each lodash utility separately, even if the main `lodash` package wouldn't be optimized. We don't have the same kinds of memory/CPU constraints that we do on the frontend, where each user has a different device with different specs.